Headers/AppLovinSDK.framework/Headers/MAAdFormat.h

```diff
@@ -0,0 +1,103 @@
+//
+//  MAAdFormat.h
+//  AppLovinSDK
+//
+//  Created by Thomas So on 8/10/18.
+//  Copyright © 2020 AppLovin Corporation. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * This class defines a format of an ad.
+ */
+@interface MAAdFormat : NSObject
+
+/**
+ * Represents a 320×50 banner advertisement.
+ */
+@property (nonatomic, strong, readonly, class) MAAdFormat *banner;
+
+/**
+ * Represents a 300×250 rectangular advertisement.
+ */
+@property (nonatomic, strong, readonly, class) MAAdFormat *mrec;
+
+/**
+ * Represents a 728×90 leaderboard advertisement (for tablets).
+ */
+@property (nonatomic, strong, readonly, class) MAAdFormat *leader;
+
+/**
+ * Represents a cross-promo advertisement.
+ */
+@property (nonatomic, strong, readonly, class) MAAdFormat *crossPromo;
+
+/**
+ * Represents a full-screen advertisement.
+ */
+@property (nonatomic, strong, readonly, class) MAAdFormat *interstitial;
+
+/**
+ * Similar to {@link interstitial}, except that users are given a reward at the end of the advertisement.
+ */
+@property (nonatomic, strong, readonly, class) MAAdFormat *rewarded;
+
+/**
+ * Represents a fullscreen ad that the user can skip or be granted a reward upon successful completion of the ad.
+ */
+@property (nonatomic, strong, readonly, class) MAAdFormat *rewardedInterstitial;
+
+/**
+ * Represents a native advertisement.
+ */
+@property (nonatomic, strong, readonly, class) MAAdFormat *native;
+
+/**
+ * The size of the AdView format ad, or `CGSizeZero` otherwise.
+ */
+@property (nonatomic, assign, readonly) CGSize size;
+
+/**
+ * Get the adaptive banner size for the screen width (with safe areas insetted) at the current orientation.
+ *
+ * <b>Note:</b> The height is the only adaptive dimension; the width spans the screen.
+ *
+ * <b>Note:</b> Only AdMob / Google Ad Manager currently has support for adaptive banners and the maximum height is 15% the height of the screen.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/banners#adaptive-banners">MAX Integration Guide ⇒ iOS ⇒ Banners ⇒ Adaptive Banners</a>
+ */
+@property (nonatomic, assign, readonly) CGSize adaptiveSize;
+
+/**
+ * Get the adaptive banner size for the provided width at the current orientation.
+ *
+ * <b>Note:</b> The height is the only adaptive dimension; the width that you provide will be passed back to you in the returned size.
+ *
+ * <b>Note:</b> Only AdMob / Google Ad Manager currently has support for adaptive banners and the maximum height is 15% the height of the screen.
+ *
+ * @param width  The width to retrieve the adaptive banner size for.
+ *
+ * @return The adaptive banner size for the current orientation and width.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/banners#adaptive-banners">MAX Integration Guide ⇒ iOS ⇒ Banners ⇒ Adaptive Banners</a>
+ */
+ // [PLP] what are the units of 'width'?
```

Comment:
In iOS terms, it's called "points"`.

Also, relevant docs: https://dash.applovin.com/documentation/mediation/ios/getting-started/banners#adaptive-banners

Also, note, we should probably add some spacing before the H1 `Adaptive Banners` title (see screenshot).
![image](https://user-images.githubusercontent.com/5104410/118220319-44c31e80-b430-11eb-8527-0b53d0e5fe7c.png)